### PR TITLE
fix: breaking layouts after RN upgrade

### PR DESCRIPTION
**Pull request by karthickcypher**

### CODE REVIEW

File: src/containers/SendTo/enterAmount.tsx [hunk 1]
```diff
@@ -126,139 +126,137 @@ export default function EnterAmount (props) {
...
+                </CyDView>
+              </CyDView>
 
-                <DynamicView dynamic dynamicWidth fD={'row'} jC={'center'} width={80} mT={20} aLIT={'center'} >
-                  <CText dynamic fF={C.fontsName.FONT_BOLD} fS={15} color={Colors.primaryTextColor}>Send on</CText>
-                  <DynamicImage dynamic dynamicWidthFix dynamicHeightFix height={15} width={15} mL={5} mT={2} resizemode='contain'
-                    source={tokenData.chainDetails.logo_url} />
-                  <CText dynamic fF={C.fontsName.FONT_BOLD} fS={15} mL={5} color={Colors.primaryTextColor}>{tokenData.chainDetails.name}</CText>
-                </DynamicView>
+              <DynamicView dynamic dynamicWidth fD={'row'} jC={'center'} width={80} mT={20} aLIT={'center'} >
```
Comment (shan-c) [inline src/containers/SendTo/enterAmount.tsx:242]: Change this to CyDView

File: src/routes/tabStack.tsx [hunk 2]
```diff
@@ -215,13 +213,10 @@ function TabStack() {
...
             <CyDAnimatedView
               // TO REDO : TABBAR ANIMATION
-              entering={SlideInDown}
-              exiting={SlideOutDown}
-              layout={Layout.easing(Easing.ease).delay(50)}
-              className={clsx('rounded-t-[24px] shadow', {
-                'mb-[-90px]': !showTabBar,
-                'absolute bottom-[-350px]': keyboardHeight,
-                'bg-white': !isIOS(),
+              layout={Layout.springify()}
```
Comment (shan-c) [inline src/routes/tabStack.tsx:216]: Change it to easing

**Review by shan-c — CHANGES_REQUESTED** [review]

File: src/routes/tabStack.tsx (hunk 2 above)
Comment (karthickcypher) [reply src/routes/tabStack.tsx:216]: Simulator Screen Recording - iPhone 14 - 2023-09-05 at 11 33 18

File: src/containers/SendTo/enterAmount.tsx (hunk 1 above)
Comment (karthickcypher) [reply src/containers/SendTo/enterAmount.tsx]: changed

**Review by shan-c — APPROVED** [review]

**Pull request merged**